index.js

```diff
@@ -10,13 +10,19 @@ const client = new Client({ intents: [ Intents.FLAGS.GUILDS,  Intents.FLAGS.GUIL
 const regex = /\[\[([^\[][^\]]*)\]\]/g;
 const MAX_RESULTS = 25;
 const MAX_RESPONSES = 3;
+const VERSION_NUMBER = "v0.3.0";
+
+var startTime;
+var cardsFetched;
```

Comment:
Can we use `let` instead of `var` here?  I generally prefer to avoid using `var` at all in modern JS code, just cause the scoping of `let` is generally a lot cleaner.

```suggestion
let startTime;
let cardsFetched;
```

Reply:
I'm OK with this.